src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/store/rca/cluster/BaseClusterRcaTest.java

```diff
@@ -310,7 +310,10 @@ private boolean compareResourceSummary(Resource resource, HotResourceSummary res
   }
 
   private boolean compareNodeSummary(String nodeId, Resource resource, HotNodeSummary nodeSummary) {
-    if (!nodeId.equals(nodeSummary.getNodeID()) || nodeSummary.getHotResourceSummaryList().isEmpty()) {
+    if (!nodeId.equals(nodeSummary.getNodeID().toString())) {
+      return false;
```

Comment:
nit: why split this if we aren't distinguishing between the two cases?

Reply:
It was just that I was debugging and it makes it easier to put breakpoints to know which branch is returning the error. :)